app/components/contacts/AllContacts.js

```diff
@@ -69,7 +69,7 @@ class AllContacts extends Component<Props, State> {
         </SearchRow>
         <ListsWrapper isModalMode={isModalMode}>
           <div>
-            {lastUsedAddresses && lastUsedAddresses.length && (
+            {lastUsedAddresses && lastUsedAddresses.length ? (
```

Comment:
what is the reason for changing this?

Reply:
As it was before, the number 0 was showing, instead of no render at all.